src/apps/content-editor/src/app/views/ItemEdit/components/Header/ItemVersioning/ScheduleFlyout/ScheduleFlyout.js

```diff
@@ -200,14 +200,12 @@ export default class ScheduleFlyout extends Component {
                   />
                 </div>
                 <div className={styles.Row}>
-                  <FieldTypeDate
-                    type="date"
+                  <FieldTypeDateTime
                     name="publish"
                     label="Publish date and time"
-                    future={true}
                     value={this.state.selectedTime}
-                    datatype={"datetime"}
                     onChange={this.handleChangePublish}
+                    disablePast
```

Comment:
Interesting. Adding note for myself to revisit and view this behavior in app. Should be fine but it is a change.

Reply:
Yep it matches the behavior of future={true} which avoids the selection of a past value

Reply:
Correct. Im assuming the rational with future={true} was that the component can only select future values.